sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -752,16 +752,14 @@ object RapidsConf {
           "happens when there are a large number of columns and relatively few " +
           "of them are being read on a large number of files. " +
           "This provides the ability to use a different path to parse and filter the footer. " +
-          "JAVA is the default and should match closely with Apache Spark. NATIVE will parse and " +
-          "filter the footer using C++. In the worst case this can be slower than JAVA, but " +
-          "not by much if anything. This is still a very experimental feature and there are " +
-          "known bugs and limitations. It should work for most cases when reading data that " +
-          "complies with the latest Parquet standard, but can run into issues for older data " +
-          "that does not fully comply with it.")
+          "AUTO is the default and decides which path to take using a heuristic. JAVA " +
+          "follows closely with what Apache Spark does. NATIVE will parse and " +
+          "filter the footer using C++. In the worst case this can be slower than JAVA. " +
+          "This is why a heruistic is used to select the appropriate parser.")
```

Comment:
Typo at "heruistic".  I'd recommend removing the last sentence, as I don't think it adds any value after it's explained that the native parser isn't always faster.
```suggestion
          "filter the footer using C++. In the worst case this can be slower than JAVA.")
```